src/main/java/com/google/pubsublite/kafka/sink/PublisherFactoryImpl.java

```diff
@@ -15,25 +15,41 @@
  */
 package com.google.pubsublite.kafka.sink;
 
-import com.google.cloud.pubsublite.CloudZone;
-import com.google.cloud.pubsublite.ProjectPath;
-import com.google.cloud.pubsublite.PublishMetadata;
-import com.google.cloud.pubsublite.TopicName;
-import com.google.cloud.pubsublite.TopicPath;
+import com.google.api.gax.rpc.ApiException;
+import com.google.cloud.pubsublite.*;
```

Comment:
This may be throwing the lint error. * is not allowed.

Reply:
Doesn't seem to be the only issue. Additionally I cannot run the suggested `mvn com.coveo:fmt-maven-plugin:format`, but working on fixing the issue.

Reply:
Thank you. Sorry that this is taking a while.

Reply:
Fixed.